docs/docs/en/guide/howto/general-setting.md

```diff
@@ -14,9 +14,17 @@ of to [language](#language) control button.
 
 ## Time Zone
 
-DolphinScheduler support time zone setting. The build-in time zone is based on the server you run DolphinScheduler. You could
+DolphinScheduler support time zone setting. 
+
+Server Time Zone
+
+Use `bin/dolphinshceduler_daemon.sh` to start server, the default time zone is UTC, you could update `SPRING_JACKSON_TIME_ZONE` in `bin/env/dolphinscheduler_env.sh`, such as `export SPRING_JACKSON_TIME_ZONE=${SPRING_JACKSON_TIME_ZONE:-Asia/Shanghai}`.<br>
```

Comment:
Use `bin/dolphinshceduler_daemon.sh` to start server, the default time zone is UTC, you could update
->
The default time zone is `UTC ` when using `bin/dolphinshceduler_daemon.sh` to start the server, you could.....